templates/authorizer.py

```diff
@@ -348,12 +354,118 @@
             "Resource": [
                 Sub(
                     "arn:aws:ssm:${{AWS::Region}}:${{AWS::AccountId}}:parameter${{{param}}}".format(
-                        param=p.title
+                        param=p.title,
                     ))
                 for p in [jwt_secret_parameter]
             ],
-        }]
-    }
+        }],
+    },
+))
+
+master_key = template.add_resource(kms.Key(
```

Comment:
It's also possible to re-use the existing key, by using the Encryption Context in a condition:

https://docs.aws.amazon.com/kms/latest/developerguide/services-parameter-store.html#parameter-store-encryption-context